code/web/src/setup/routes/user.js

```diff
@@ -26,5 +27,11 @@ export default {
     path: '/user/subscriptions',
     component: Subscriptions,
     auth: true
+  },
+
+  survey: {
+    path: '/user/survey',
+    component: DummyComponent,
+    auth: true
```

Comment:
The style survey route, currently a provided dummy component is displayed, later to be replaced by our survey form. Opted for this option rather than create a new file for survey path.

Reply:
makes sense. that explains it